compose-setup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+function setup_database() {
```

Comment:
```bash
setup_database() {
```
It's recommended to declare functions without the `function` keyword for portability reasons.